python-lib/lal/config/dss_parameter.py

```diff
@@ -49,6 +49,8 @@ def format_err_mesg(self, parameter):
         return f'Error for parameter "{parameter.name}" - {formatted_err_msg}'
 
     def _exists(self, value):
```

Comment:
where is this method used?

Reply:
It is used if DSSParameter has param required set to True.